$MirOS: ports/www/dillo/v2/patches/patch-src_dns_c,v 1.2 2010/02/21 20:08:43 bsiegert Exp $
--- src/dns.c.orig	Mon Jun 22 20:02:15 2009
+++ src/dns.c	Mon Aug 24 16:22:27 2009
@@ -24,6 +24,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
+#include <unistd.h>
 
 #include "msg.h"
 #include "dns.h"
